app/scripts/modules/core/src/entityTag/notifications/NotificationList.tsx

```diff
@@ -17,15 +17,14 @@ export interface INotificationListProps {
 export class NotificationList extends React.Component<INotificationListProps> {
   public render() {
     const { notifications, onEditTag, onDeleteTag } = this.props;
-
     return (
       <div className="notification-list">
         {notifications.map((notification: INotification, idx: number) => {
           const {
             entityTag: {
+              lastModified,
               value: { title, message, tagline },
             },
-            entityTags: { lastModified },
```

Comment:
Safe to delete?

Reply:
yeah, it should track each tag rather than all the tags of the entity